src/main/frontend/handler/editor.cljs

```diff
@@ -375,24 +376,14 @@
   ([block value
     {:keys [force?]
      :as opts}]
-   (let [{:block/keys [uuid page format repo content properties]} block
+   (let [{:block/keys [page format repo content]} block
          repo (or repo (state/get-current-repo))
          format (or format (state/get-preferred-format))
          page (db/entity repo (:db/id page))
-         block-id (when (map? properties) (get properties :id))
          content (-> (property/remove-built-in-properties format content)
                      (drawer/remove-logbook))]
-     (cond
-       (another-block-with-same-id-exists? uuid block-id)
```

Comment:
Because of changes to this PR. 
But It's only safe to remove this check while deleting a target block that has been referenced. I made the change in https://github.com/logseq/logseq/pull/9406/commits/aa7cd9eefb345abd72f3fcd17d63fd6e8edca71b.